adapters/amqp-vertx/src/test/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapterWithoutClientTest.java

```diff
@@ -0,0 +1,270 @@
+package org.eclipse.hono.adapter.amqp;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.qpid.proton.amqp.messaging.AmqpValue;
+import org.apache.qpid.proton.message.Message;
+import org.eclipse.hono.client.HonoClient;
+import org.eclipse.hono.client.MessageSender;
+import org.eclipse.hono.client.RegistrationClient;
+import org.eclipse.hono.client.TenantClient;
+import org.eclipse.hono.config.ProtocolAdapterProperties;
+import org.eclipse.hono.service.command.CommandConnection;
+import org.eclipse.hono.util.RegistrationConstants;
+import org.eclipse.hono.util.ResourceIdentifier;
+import org.eclipse.hono.util.TenantConstants;
+import org.eclipse.hono.util.TenantObject;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.proton.ProtonConnection;
+import io.vertx.proton.ProtonDelivery;
+import io.vertx.proton.ProtonReceiver;
+import io.vertx.proton.ProtonServer;
+
+/**
+ * Verifies the behaviour of {@link VertxBasedAmqpProtocolAdapter}. In this test setup, the AMQP adapter is not deployed
+ * as a running Verticle and does not use an AMQP client to simulate a client device.
+ * Rather, the setup manually instantiates the adapter and gives it a user defined ProtonServer.
+ */
+@RunWith(VertxUnitRunner.class)
+public class VertxBasedAmqpProtocolAdapterWithoutClientTest {
```

Comment:
This seems to be the unit tests, right? If so, can we please simply name this class `VertxBasedAmqpProtocolAdapterTest`?

Reply:
yes, it is. i rename it to distinguish it from the other test. will change it now